flag_generated.go

```diff
@@ -30,6 +31,22 @@ func (f BoolFlag) GetName() string {
 	return f.Name
 }
 
+// TakesValue returns true of the flag takes a value, otherwise false
+func (f BoolFlag) TakesValue() bool {
+	return false
+}
+
+// GetUsage returns the usage string for the flag
+func (f BoolFlag) GetUsage() string {
+	return f.Usage
+}
+
+// GetValue returns the flags value as string representation and an empty
+// string if the flag takes no value at all.
+func (f BoolFlag) GetValue() string {
+	return ""
+}
```

Comment:
⭐⭐⭐ _(must change)_

It looks like you edited this directly? What you'll want to do here is change the `generate-flag-types` file.

And that file is going to be removed in this PR => https://github.com/urfave/cli/pull/836 which means that this PR now depends on that one ^^

Reply:
https://github.com/urfave/cli/pull/836 has been merged! ✔️

Reply:
Alright I will change it